src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -308,6 +308,7 @@ public static class Release
                 // Keep alphabetical
                 //
                 public static readonly string AgentReleaseDirectory = "agent.releaseDirectory";
+                public static readonly string AgentReleaseRootDirectory = "agent.releaseRootDirectory";
```

Comment:
Yes that's pretty much I am looking for. But the file that gets created and queued for upload is the last action the task. Post this the file is still waiting for to be uploaded. But in the meanwhile job completion goes ahead and deletes the entire TEMP directory.. any idea how to handle that?